### ISSUE TITLE
Page for Carnatic Scores

### ISSUE THREAD

**Issue by ananthp** [ananthp/carnatic_scores#5]
A page will be easier to go thru and download scores for the end user. Done. See: http://ananthp.github.io/carnatic_scores/

**Comment by VynM** [comment]
I couldn't find a good place to write this comment, so I'm doing so here. Firstly, let me say, great work! And I hope you keep it up.

Now, the main reason for my comment. I wanted to ask whether you'd consider rewriting (or writing an alternative version) the scores with the sahityam transilterated using the Harvard-Kyoto convention (the best system for accurate transliteration of Indian languages using Roman letters, in my opinion). For example, see [Thyagaraja Vaibhavam](http://thyagaraja-vaibhavam.blogspot.in/2009/03/tyagaraja-kritis-alphabetical-list.html) — another great repository (of only sahityam and meaning), which you must already have seen.

I don't mean that you should personally do this work. I see that you already have different versions of some scores in the repository. So, if you're interested, I'd be happy to write these new HK versions to add to the repo.

**Comment by ananthp** [comment]
Thanks for the feedback. I need to integrate a comments/feedback system to the site.

I started writing the scores for my personal use, so chose the spelling style that's closer to plain intuitive transliteration. My focus was on the notation part, where I've evolved a style, breaking away from traditional box like chunks of notes, divided by thalam counts. For sahityam part, I've tried to avoid any glaring mistakes, strived to show words and musical phrasing clearly whenever possible. 

If you prefer Harvard-Kyoto style in sahityam feel free to clone the repo, modify, and send me a pull request. The scores are creative commons licensed, I've used only open source tools to create it -- Inkscape and FreeSerif/FreeSans fonts. So feel free to customize.

Need any help? Tweet to @ananth
